src/components/page-layout/NavigationBar.js

```diff
@@ -1,47 +1,83 @@
 import React from 'react';
 
 // Componenets
-import { Button, Grid } from '@material-ui/core';
+import {
+  Button,
+  Grid,
+  Typography,
+} from '@material-ui/core';
 import { Link } from 'gatsby';
-import StorefrontIcon from '@material-ui/icons/Storefront';
 import ShoppingBasketIcon from '@material-ui/icons/ShoppingBasket';
-import HomeIcon from '@material-ui/icons/Home';
+import ecoLogo from '../../images/EcoCart_Logo.png';
 
 // Styles
 import styles from './styles/navigation-bar.module.css';
 
-const NavigationBar = ({ toggleModal }) => (
-  <div className={styles.container}>
-    <Grid container justify="space-between">
-      <Grid item>
-        <div>
+const NavigationBar = ({ toggleModal, resetHeaderTop }) => (
+    <Grid
+      container
+      justify="center"
+      alignItems="center"
+      className={styles.container}
+    >
+    <image src='public/EcoCart_Log.png' alt="Girl in a jacket" width="500" height="600" />
+      <Grid
+        item
+        sm={1}
+      >
         <Link
           to='/'
         >
           <Button
             className={styles.button}
           >
-            <HomeIcon className={styles.icon} />
+            <img
+              src={ecoLogo}
+              alt='EcoCart'
+              className={styles.ecoLogo}
+            />
+          </Button>
+        </Link>
+      </Grid>
+      <Grid
+        item
+        sm={5}
+      >
+        <Link
+          to='/'
+          style={{ textDecoration: 'none' }}
```

Comment:
This is static and reused on line 59, so could be pulled out into the css module